contribs/gnofaucet/gh.go

```diff
@@ -19,6 +19,33 @@ import (
 // http and RPC GitHub middleware handlers
 const ghUsernameKey = "gh-username"
 
+// claimRPCMethod is a method that is exactly like the default drip method but without any amount set
+const claimRPCMethod = "claim"
+
+// getClaimRPCMethod is a method that return the amount that the user will be able to claim
+const getClaimRPCMethod = "getClaim"
```

Comment:
What do you think about renaming this to `checkClaim`, so we avoid confusion with `claim`?